cli/remotebazel/remotebazel.go

```diff
@@ -33,6 +35,8 @@ const (
 )
 
 var (
+	execOs            = flag.String("os", "", "If set, requests execution on a specific OS.")
+	execArch          = flag.String("arch", "", "If set, requests execution on a specific CPU architecture.")
```

Comment:
can we make linux x86_64 the flag default here since this is what we fully support? This will allow easily doing linux builds from mac and make this work out of the box more easily.

If/when we support mac and arm and other stuff we can flip the defaults back to the client's arch/os.

Reply:
Done.